src/WorkBC.Web/ClientApp/projects/jb-lib/src/lib/filters/services/filter.service.ts

```diff
@@ -585,6 +585,9 @@ export class FilterService extends BaseService {
     if (pos > -1) {
       mainFilters.removeActiveFilter(filter);
 
+      //reset to page 1 when any filter is changed
+      mainFilters.pagination.currentPage = 1;
```

Comment:
Single-line code change to reset the pagination to page 1 when any filter is removed